### DISCUSSION TITLE
Hard to get a count of discussion posts

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#46062] [Mobile] [answered]
GitHub Mobile beta

Consider:
https://github.com/community/community/discussions/categories/mobile?discussions_q=author%3Ajsoref+category%3AMobile

No matter how many pages of posts I have, I can count the number of posts on the first page, jump to the last page, count the number of complete pages and then count the number of items on the list page.

With Mobile, I have infinite scrolling and just have to carefully keep track. ..

Mobile with infinite scroll
![Screenshot_20230201-172708.png](https://user-images.githubusercontent.com/2119212/216178057-0612be87-bd3b-4e79-b52b-09817b4213c4.png)

Chrome mobile view with pagination 
![Screenshot_20230201-172633.png](https://user-images.githubusercontent.com/2119212/216177944-2e70ac2e-a3b8-44dd-9898-b91dfbfc5446.png)

.

**Comment by alcere** [comment, marked as answer]
Hi @jsoref, thanks for sharing this feedback! I have shared your request with the team, and the idea will be evaluated.
